### DISCUSSION TITLE
seeker/r2c2 blenderbot 3B/model train

### DISCUSSION THREAD

**Discussion by jacek-michalak** [facebookresearch/ParlAI#4885] [General]
Suppose I want to add to the model: seeker/r2c2 blenderbot 3B/model knowing that: -2+2=0 (single record). What files do I need to create to train the above model and what will the training command look like?

**Comment by klshuster** [comment]
I suggest looking at the [tutorials for adding tasks](https://parl.ai/docs/tutorial_task.html) to ParlAI, and then subsequently the [seeker project page](https://parl.ai/projects/seeker/) for how to train such a model
